packages/contracts/contracts/voting/majority/IMajorityVoting.sol

```diff
@@ -180,35 +195,41 @@ interface IMajorityVoting {
     /// @return The participation value.
     function participation(uint256 _proposalId) external view returns (uint256);
 
-    /// @notice Returns all information for a proposal by its ID.
+    /// @notice Returns the vote mod () stored in the plugin settings.
```

Comment:
`mode`

Reply:
Fixed in ef7fed8d7b6b3dc5c7928f745fc1c6433a30fecd